tests/ecode/test_sahp.py

```diff
@@ -0,0 +1,144 @@
+"""bluepyefe.ecode.sAHP tests"""
+
+import unittest
+import glob
+import json
+
+import bluepyefe.extract
+import bluepyefe.tools
+from tests.utils import download_sahp_datafiles
+
+
+def get_sahp_config(absolute_amplitude=False):
+    download_sahp_datafiles()
+
+    interesting_efeatures = {
+        "Spikecount": {},
+        "mean_frequency": {},
+        "ISI_CV": {},
+        "AP1_amp": {},
+        "AP_width": {},
+    }
+
+    files_metadata1 = []
+    for file in glob.glob("./tests/exp_data/X/X_sAHP_ch0_*.ibw"):
+        files_metadata1.append(
+            {
+                "i_file": file,
+                "v_file": file.replace("ch0", "ch1"),
+                "i_unit": "A",
+                "v_unit": "V",
+                "t_unit": "s",
+                "dt": 0.00025,
+                "ljp": 14,
+                "ton": 10,
```

Comment:
It looks like dt is in [s], but ton, tmid, tmid2 and toff are in [ms]? If this is the way we are supposed to give the data, we should be more explicit about it in the documentation, because users will have a hard time figuring this out

Reply:
I agree, I converted in ms to make it less confusing and added a comment for the ton, tmid, tmid2 and toff

Reply:
wait, I am confused now. You changed the units without changing the data? Do we even know what are the units used in the data files we use in the tests?

Reply:
but if you adjust the dt accordingly it shouldn't matter, no? the dt is correct based on the data

Reply:
But does the extraction give the same values? I mean, if we change dt from `0.00025` to `0.25`, the output of the extraction should also change, no?

Reply:
but if we change t_unit accordingly

Reply:
Oh, you also changed the units. I did not see it. Sorry for dragging this discussion.

Reply:
no worries, it was easy to miss